src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/uat/common/SuccessArgumentsProvider.java

```diff
@@ -0,0 +1,44 @@
+package uk.nhs.digital.nhsconnect.lab.results.uat.common;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.provider.Arguments;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class SuccessArgumentsProvider extends
+    uk.nhs.digital.nhsconnect.lab.results.uat.common.AbstractArgumentsProvider {
```

Comment:
Can we import this so we don't need the fully qualified class name here?

Reply:
It's actually already in the same package, so no import needed. Thanks for the catch though!